db/migrate/20160628233646_create_posts.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePosts < ActiveRecord::Migration
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text   :body
+      t.integer :post_id 
```

Comment:
Rails will automatically generate a primary key. You don't need to do that.

You do, however, need to create a column for any foreign keys. That's not necessary for posts though...